architecture/index.adoc

```diff
@@ -44,6 +44,8 @@ as they flow through your system
 
 .{product-title} Architecture Overview
 image::architecture_overview.png[{product-title} Architecture Overview]
+
+For more information on the nodes in the architecture overview see xref:infrastructure_components/kubernetes_infrastructure.adoc#node[Nodes].
```

Comment:
To also capture "master nodes" in this pointer, maybe the following tweak, linking to the top of the page:

"For more information on the node types in the architecture overview, see xref:infrastructure_components/kubernetes_infrastructure.adoc#architecture-infrastructure-components-kubernetes-infrastructure[Kubernetes Infrastructure]."